apps/condo/domains/user/gql.js

```diff
@@ -36,7 +36,7 @@ const SIGNIN_MUTATION = gql`
 
 const CHANGE_PASSWORD_WITH_TOKEN_MUTATION = gql`
     mutation changePasswordWithToken($data: ChangePasswordWithTokenInput!) {
-        result: changePasswordWithToken(data: $data) { status, email }
+        result: changePasswordWithToken(data: $data) { status, phone }
```

Comment:
you can add here phone and email

Reply:
we do not use email as a result - we need a phone to authenticate the user after changing the password